packages/core/cli/src/utils/getIntegrationsFromPackage.ts

```diff
@@ -0,0 +1,4 @@
+export default (): string[] => [
```

Comment:
AYC shouldn't be available yet (its not working correctly). I'd put commercetools only for now until we check if shopify is working, also filename is not accurate anymore ;)